app/src/main/java/com/kickstarter/viewmodels/PledgeFragmentViewModel.kt

```diff
@@ -883,9 +883,9 @@ interface PledgeFragmentViewModel {
                     .compose<Pair<Double, Double>>(combineLatestPair(selectedPledgeAmount))
                     .map { it.first - it.second }
 
-            val pledgeMaximumIsGone = pledgeMaximum
+            val pledgeMaximumIsGone = currencyMaximum
                     .compose<Pair<Double, Double>>(combineLatestPair(total))
-                    .map { it.first > it.second }
+                    .map { it.first >= it.second }
                     .distinctUntilChanged()
 
             pledgeMaximumIsGone
```

Comment:
LGTM!

Reply:
Do we need to update tests for this change?

Reply:
Don't think so, we currently don't have tests for Bonus support. I can add a ticket to add them in the future

Reply:
we do have test for the PledgeMaximum is gone in PledgeFragmentViewModelTest. If all of them are passing that's nice :)